Makefile

```diff
@@ -1,15 +1,15 @@
-CC=g++ -std=c++14
+CC=g++
 src=./src/
 include=-I./include/
 geo=./src/geometry/
 bin=./bin/
 bld=./build/
-flags=-std=c++14 -fopenmp
+flags=-std=c++14 -openmp
```

Comment:
Otherwise setting `alias g++="clang"` would fix it on Ubuntu but that's just wrong :D